hudi-common/src/main/java/org/apache/hudi/common/table/log/BaseHoodieMergedLogRecordScanner.java

```diff
@@ -210,7 +210,7 @@ protected void processNextDeletedRecord(DeleteRecord deleteRecord) {
     String key = deleteRecord.getRecordKey();
     HoodieRecord oldRecord = records.get(key);
 
-    Comparable deleteOrderingVal = deleteRecord.getOrderingValue() == null ? orderingFieldDefault :  deleteRecord.getOrderingValue();
+    Comparable deleteOrderingVal = deleteRecord.getOrderingValue() == null ? orderingFieldDefault : deleteRecord.getOrderingValue();
```

Comment:
we can call getOrderingVal(orderingFieldDefault) now

Reply:
Done.